.github/workflows/ibctest.yml

```diff
@@ -70,8 +72,9 @@ jobs:
 
       - name: ibctest
         run: make ibctest-multiple
-  path-filter:
-    runs-on: ubuntu-latest
+
+  scenarios:
+    runs-on: self-hosted
```

Comment:
Self hosted runs a bit faster. 6m vs. 10m.

Reply:
Since we currently only have one self-hosted runner for the relayer and a self-hosted runner can only run one test at a time, 10m on the gh runner might still result in a faster overall test suite execution.

Reply:
In my single test (granted it was only 1) it seemed to run faster on the self-hosted. But I'm happy to change it back.